config/locales/kaminari.fr.yml

```diff
@@ -2,8 +2,14 @@
 fr:
   views:
     pagination:
+      aria_next: "Next page"
+      aria_previous: "Previous page"
       first: "&laquo; Premier"
```

Comment:
Just as a reading note - the `fr.yml` file has more strings than the `en` version because these files are overrides for `kaminari` defaults. By not providing a given key in `en.yml` we let `kaminari` use the default (this confused me initially so thought I'd mention here!)